pkg/webhooks/webhookpatch.go

```diff
@@ -167,3 +177,31 @@ func (w *WebhookCertPatcher) patchMutatingWebhookConfig(
 	_, err = client.Update(context.TODO(), config, metav1.UpdateOptions{})
 	return err
 }
+
+// startCaBundleWatcher listens for updates to the CA bundle and patches the webhooks.
+func (w *WebhookCertPatcher) startCaBundleWatcher(stop <-chan struct{}) {
+	watchCh := w.CABundleWatcher.AddWatcher()
+	options := metav1.ListOptions{LabelSelector: fmt.Sprintf("%s=%s", label.IoIstioRev.Name, w.revision)}
+	for {
+		select {
+		case <-watchCh:
+			whcList, err := w.client.AdmissionregistrationV1().MutatingWebhookConfigurations().List(context.TODO(), options)
```

Comment:
runWebhookController is already creating a watch for MWHs matching this query. We should use that to avoid extra calls to API server?

Reply:
Thanks, reuse now.